src/languages/en-US/commands/fun.json

```diff
@@ -184,6 +184,7 @@
 	"loveResult": "Result",
 	"markovTimer": "Processed in {{timer}}.",
 	"markovNoMessages": "The channel or user has no messages.",
+	"markovNsfwChannel": "I am sorry, but you cannot use the markov command on an NSFW channel here.",
```

Comment:
describing *why* the user can't use the command in a NSFW channel, rather then just telling them they can't - is a much better error message imo.

Reply:
Resolved in https://github.com/skyra-project/skyra/pull/1607/commits/6530a10b919464a2f36612ca8d7d530546f0dace